app/javascript/state/features/index.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import type { InitialAction } from '../reducers';
+import type { AppState } from '../';
+import type { Store } from 'redux';
+
+export type State = {
+  googlepay: boolean,
+};
+
+const defaults: State = {
+  googlepay: false,
+};
+
+type Action =
+  | InitialAction
+  | { type: '@@chmp:feature:enable', featureName: string }
+  | { type: '@@chmp:feature:disable', featureName: string };
+
+export default function reducer(s: State = defaults, a: Action): State {
```

Comment:
What does this do?

Reply:
This file is the redux store for feature flags. Basically, state is a key/value map which, for now, just has googlepay as a feature flag. The default value is:

```js
{
  googlepay: false
}
```

This PR adds functionality to enable/disable feature flags (which simply updates this store). To enable or disable features, you can do:

```js
window.champaign.features.enable('featurename');
window.champaign.features.disable('featurename');
```

The `FeaturesHelper` class below is the constructor for that functionality. You pass a redux store to it, and you can list, enable, disable, or check if a feature is enabled:

```js
const features = new FeaturesHelper(store);
features.list();
// Will return an array of features, at the moment only ["googlepay"].

features.enable('googlepay');
features.disable('googlepay');

// To check if a feature is enabled: 
features.isEnabled('googlepay');
```